.bower-postinstall.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+BOWER_DIR=$1
+
+if [ $BOWER_DIR ]; then
+
+  # Hopscotch - Rename .css file to .scss
+  HOPSCTOCH_DIR=$BOWER_DIR/hopscotch/dist/css
+  HOPSCTOCH_FILE=$HOPSCTOCH_DIR/hopscotch.css
+  if [ -e "$HOPSCTOCH_FILE" ]; then
+    mv "$HOPSCTOCH_FILE" "$HOPSCTOCH_DIR/_hopscotch.scss"
+  fi
+
+  # MTP Common - copy shared gulp tasks
+  TASKS_DIR="./tasks"
+  SHARED_TASKS_DIR="$TASKS_DIR/common"
+  if [ ! -e "$TASKS_DIR" ]; then
+    mkdir "$TASKS_DIR"
+  fi
+  if [ -e "$SHARED_TASKS_DIR" ]; then
+    rm -rf "$SHARED_TASKS_DIR"
+    mkdir "$SHARED_TASKS_DIR"
+  else
+    mkdir "$SHARED_TASKS_DIR"
+  fi
+  cp -vr "$BOWER_DIR/money-to-prisoners-common/tasks/" "$SHARED_TASKS_DIR"
```

Comment:
I'm not sure how I feel about this. For the gulp tasks, wouldn't it be better to [have gulp require the tasks from `common`](https://github.com/gulpjs/gulp/issues/438) rather than copying them into our heirarchy? I worry that this way lies debugging weirdness and confusion about what actually lives where.